signac/contrib/utility.py

```diff
@@ -188,6 +267,29 @@ def _mkdir_p(path):
 
 
 def split_and_print_progress(iterable, num_chunks=10, write=None, desc='Progress: '):
+    """Split the progress and prints it.
+
+    Parameters
+    ----------
+    iterable : list
+        list of state points.
+    num_chunks : int
+        (Default value = 10)
+    write :
+        (Default value = None)
+    desc : str
+        (Default value = 'Progress: ')
```

Comment:
Add a short description for these variables.

Reply:
added, please have a look.

Reply:
Looks good!